pkgs/development/tools/rust/cargo-rr/default.nix

```diff
@@ -9,16 +9,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-rr";
-  version = "0.1.3";
+  version = "v0.2.0";
 
   src = fetchFromGitHub {
     owner = "danielzfranklin";
     repo = pname;
     rev = version;
-    sha256 = "01m8fdz9as2fxnzs9csvbc76qxzbb98a66dh7w4a5q855v38g0zy";
+    sha256 = "sha256-lQS+bp1u79iO8WGrkZSFEuonr1eYjxIQYhUvM/kBao4";
   };
 
-  cargoSha256 = "0fjs76n6bbbv83s213h2dgsszgxy4hbjsclyk9m81b3bfbmmb9sa";
+  cargoSha256 = "sha256-EkF4BRoV5rR+u5Vo+JlwsnpSpcR600oKbYZU9teg1P0=";
 
   passthru = {
     updateScript = genericUpdater {
```

Comment:
Maybe fix the version here too? Not sure.

Reply:
Replaced to a gitUpdater so it has support for the prefix.